include/deal.II/base/vectorization.h

```diff
@@ -588,7 +576,19 @@ class VectorizedArray<double>
   /**
    * This gives the number of vectors collected in this class.
    */
-  static const unsigned int n_array_elements = 8;
+  static constexpr unsigned int n_array_elements = 8;
+
+  /**
+   * Default constructor
+   */
+  VectorizedArray() = default;
+
+  /**
+   * Conversion constructor from a double
+   */
+  DEAL_II_ALWAYS_INLINE explicit VectorizedArray(const double x)
+    : data{_mm512_set1_pd(x)}
```

Comment:
Because I changed the constructors for Tensor<rank, dim, Number>, they now call the one-parameter constructor VectorizedArray(double) instead of the assignment operator.

Based on what I remember, when I added the one-parameter constructor, the compiler demanded me to add a default constructor as well.

The generic VectorizedArray<Number> defined in the beginning, on the other hand, didn't need any constructor because it does not need to call any functions like _mm512_set1_pd.